### DISCUSSION TITLE
does librdkafka support ApiKey DescribeLogDirs

### DISCUSSION THREAD

**Discussion by en-vee** [edenhill/librdkafka#4737] [Q&A]
As the title says, is this API key supported by the admin client ?
Any plans of adding this feature, if not already supported ?

**Comment by baac0** [comment]
According to the documentation (https://github.com/confluentinc/librdkafka/blob/master/INTRODUCTION.md#supported-protocol-versions), it is not yet supported. However, I, too,  would be interested to learn if it is planned to add this API key as well as 
DescribeClientQuota (48)/AlterClientQuotas (49) to librdkafka in the foreseeable future?

Regards, Achim
